### DISCUSSION TITLE
Extension Builder for Typo3 10.4 LTS

### DISCUSSION THREAD

**Discussion by klabauterKoffer** [FriendsOfTYPO3/extension_builder#412] [Q&A] [answered]
How do I find the latest development status of the extension Builder for TYPO3 10.4.
Can I download it directly from the project version control system?
As described in the book Practical Knowledge Typo3 10 LTS ..

**Comment by simonschaufi** [comment, marked as answer]
The recommended way is to use composer and require the eb like this `friendsoftypo3/extension-builder:dev-master` until there is a final release. If you don't use composer in your project, either switch to composer or download the extension from github and put it in `typo3conf/ext` like any other extension.

**Reply by simonschaufi** [reply]
There is now an official release so you can just require the extension  like so: `friendsoftypo3/extension-builder`
